lab15-jacob/challenge/challenge.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
```

Comment:
This is totally fine, though could have just been written as a method on the Doubly Linked List prototype as well, which would have cut down some of the code you wrote. ;-)